Makefile

```diff
@@ -98,6 +109,10 @@ cli-dev:
 	@echo "==> Installing consul-k8s CLI tool for ${GOOS}/${GOARCH}"
 	@cd cli; go build -o ./bin/consul-k8s; cp ./bin/consul-k8s ${GOPATH}/bin/
 
+cli-fips-dev:
```

Comment:
This doesn't build for me on my M1. Do I need to install something?

Reply:
@skpratt I do not get any output at all, just the 'installing CLI' message. The binary isn't built for me.

Reply:
gcc version would be it!